packages/frontend-api/src/Resources/config/graphql-types/ModelType/Order/OrderPaymentsConfig.types.yaml

```diff
@@ -4,8 +4,8 @@ OrderPaymentsConfigDecorator:
     config:
         fields:
             currentPayment:
-                type: "Payment!"
+                type: "Payment"
```

Comment:
Is it worth updating the description? I would add something like "null when the original payment is not available anymore due to the reached limit of max payment transactions count" - while typing it, new questions raised in my head :smile: What happens when the current payment is not available anymore? (e.g. is hidden or deleted) But I think it was discussed during the original implementation as unlikely to happen scenario :shrug:

Reply:
question passed to @pk16011990 :grin: